torchgeo/datasets/eurocrops.py

```diff
@@ -0,0 +1,285 @@
+        ("DE_NRW_2021.zip", 2021, "a5af4e520cc433b9014cf8389c8f4c1f"),
+        ("DK_2019.zip", 2019, "d296478680edc3173422b379ace323d8"),
+        ("EE_2021.zip", 2021, "a7596f6691ad778a912d5a07e7ca6e41"),
+        ("ES_NA_2020.zip", 2020, "023f3b397d0f6f7a020508ed8320d543"),
+        ("FR_2018.zip", 2018, "282304734f156fb4df93a60b30e54c29"),
+        ("HR_2020.zip", 2020, "8bfe2b0cbd580737adcf7335682a1ea5"),
+        ("LT_2021.zip", 2021, "c7597214b90505877ee0cfa1232ac45f"),
+        ("LV_2021.zip", 2021, "b7253f96c8699d98ca503787f577ce26"),
+        ("NL_2020.zip", 2020, "823da32d28695b8b016740449391c0db"),
+        ("PT.zip", 2021, "3dba9c89c559b34d57acd286505bcb66"),
+        ("SE_2021.zip", 2021, "cab164c1c400fce56f7f1873bc966858"),
+        ("SI_2021.zip", 2021, "6b2dde6ba9d09c3ef8145ea520576228"),
+        ("SK_2021.zip", 2021, "c7762b4073869673edc08502e7b22f01"),
+        # Year is unknown for Romania portion (ny = no year).
+        # ("RO_ny", ??),
+    ]
+
+    # Color palette to choose from.
+    # There are hundreds of classes so we pick color via modulo of class index.
+    colors = [
+        (70, 107, 159, 255),
+        (209, 222, 248, 255),
+        (222, 197, 197, 255),
+        (217, 146, 130, 255),
+        (235, 0, 0, 255),
+        (171, 0, 0, 255),
+        (179, 172, 159, 255),
+        (104, 171, 95, 255),
+        (28, 95, 44, 255),
+        (181, 197, 143, 255),
+        (204, 184, 121, 255),
+        (223, 223, 194, 255),
+        (220, 217, 57, 255),
+        (171, 108, 40, 255),
+        (184, 217, 235, 255),
+        (108, 159, 184, 255),
+    ]
+
+    def __init__(
+        self,
+        paths: Union[str, Iterable[str]] = "data",
+        crs: Optional[CRS] = None,
+        res: float = 0.00001,
+        transforms: Optional[Callable[[dict[str, Any]], dict[str, Any]]] = None,
+        download: bool = False,
+        checksum: bool = False,
+        hcat_level: int = 6,
+    ) -> None:
+        """Initialize a new Dataset instance.
+
+        Args:
+            paths: one or more root directories to search for files to load
+            crs: :term:`coordinate reference system (CRS)` to warp to
+                (defaults to the CRS of the first file found)
+            res: resolution of the dataset in units of CRS
+            transforms: a function/transform that takes an input sample
+                and returns a transformed version
+            download: if True, download dataset and store it in the root directory
+            checksum: if True, check the MD5 of the downloaded files (may be slow)
+            hcat_level: use labels at this level of the
+                `HCATv2 hierarchy <https://www.eurocrops.tum.de/taxonomy.html>`__.
+                Valid levels are 3, 4, 5, and 6.
+
+        Raises:
+            DatasetNotFoundError: If dataset is not found and *download* is False.
+
+        .. versionchanged:: 0.5
+           *root* was renamed to *paths*.
```

Comment:
You can remove this. This dataset was not added until 0.6, so the `root` parameter was never renamed for this dataset.